requirements.txt

```diff
@@ -0,0 +1,4 @@
+youtube_dl
```

Comment:
I committed that file at some point in time during debugging. blunder spotted :100: really embarrassed :p